app/mailers/devise_custom_mailer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Mailer Overrides for Devise default emails
+class DeviseCustomMailer < Devise::Mailer
+  include Devise::Controllers::UrlHelpers # Optional. eg. `confirmation_url`
```

Comment:
why is this needed?

Reply:
It probably isn't.  Was the default from a Custom Mailer template.  I'm gonna leave it until we've finished all the other mailers though, just in case we end up needing them.